lib/statsd/instrument/backends/udp_backend.rb

```diff
@@ -24,7 +26,7 @@ class DogStatsDProtocol
       SUPPORTED_METRIC_TYPES = BASE_SUPPORTED_METRIC_TYPES.merge(h: true, _e: true, _sc: true, d: true)
 
       def generate_packet(metric)
-        packet = ""
+        packet = StringIO.new
```

Comment:
`StringIO` is a bit much, just do `packet = +""`

Reply:
TIL!